src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/EventListener.java

```diff
@@ -43,9 +45,9 @@
 import hudson.model.ParametersDefinitionProperty;
 import jenkins.model.Jenkins;
 import jenkins.model.ParameterizedJobMixIn;
+import org.apache.commons.lang.StringUtils;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
-import org.apache.commons.lang.StringUtils;
 import org.slf4j.Logger;
```

Comment:
Not needed

Reply:
Hi @panicking ,
this is a auto sort by IDEA, not delete this import.
Could we keep it?

Reply:
Hi @panicking ,
Agree with you that only push functionality related code.

But I think the problem is here that usually we have auto import on fly in IDEA to avoid import them by hand.
Would you like to share the rules of import for this repo? 
Thanks.

Reply:
@evoToBetter Yes I understand but when you add what to be included on the patch you can remove this from them without add the specific line of code using idea or git add -p. I'm using vim to work on this plugin or vscode, I have specific vscode workspace for jenkins plugin